server/src/test/java/org/opensearch/search/aggregations/startree/RangeAggregatorTests.java

```diff
@@ -174,6 +174,7 @@ public void testRangeAggregation() throws IOException {
             }
         }
 
+        ir.close();
```

Comment:
Can you revert this change?

Reply:
Will fix with rebase once the other one is merged.